### DISCUSSION TITLE
ESUPOL Dataset

### DISCUSSION THREAD

**Discussion by melaniefumfack** [irgroup/dis25-2021#9] [Q&A]
Hallo Zusammen, 

gibt es schon Information, wo wir den ESUPOL Datensatz herbekommen? 
Oder bekommen wir den Datensatz noch? 

Danke im Voraus!

**Comment by HaakFab** [comment]
Guten Tag,

Ich habe soeben eine knappe Beschreibung der drei für Sie relevanten ESuPol Datensätze sowie Ausschnitte (Daten von 1-3 Tagen) hochgeladen. So können Sie sich einen Eindruck von den Daten verschaffen, beachten Sie aber, dass Sie für die Projektarbeit am Ende des Semesters noch einmal umfangreichere Varianten der Datensätze bekommen werden. 

Viele Grüße und ein schönes Pfingswochenende, 
Fabian Haak
